build.gradle.kts

```diff
@@ -191,6 +191,7 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-aop")
 
   implementation("org.apache.tika:tika-core:${tikaVersion}")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.3-native-mt")
```

Comment:
Could you please change for version 1.6.4 ? see [1.6.4](https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kotlinx-coroutines-core/1.6.4)